k8s/openshift/byo/baremetal/containerized_openshift_readme.md

```diff
@@ -1,5 +1,475 @@
+               Default registries searched: registry.access.redhat.com
+               Failed connecting to: registry.access.redhat.com
+
+
+  2. Hosts:    20.10.45.112
+     Play:     OpenShift Health Checks
+     Task:     Run health checks (install) - EL
+     Message:  One or more checks failed
+     Details:  check "docker_image_availability":
+               One or more required container images are not available:
+                   openshift3/node:v3.9.33,
+                   openshift3/openvswitch:v3.9.33,
+                   openshift3/ose-deployer:v3.9.33,
+                   openshift3/ose-docker-registry:v3.9.33,
+                   openshift3/ose-haproxy-router:v3.9.33,
+                   openshift3/ose-pod:v3.9.33,
+                   openshift3/registry-console:v3.9
+               Checked with: skopeo inspect [--tls-verify=false] [--creds=<user>:<pass>] docker://<registry>/<image>
+               Default registries searched: registry.access.redhat.com
+               Failed connecting to: registry.access.redhat.com
+```
+
+The following command can be used to download the required package(master and node).
+
+```
+docker pull openshift3/node:v3.9.33
+```
+
+-While installation if you get error "Currently, NetworkManager must be installed and enabled prior to installation" then follow the below mentioned steps to make it active(master and node).
+
+```
+[root@ocp-node-3 ~]# systemctl show NetworkManager | grep ActiveState
+ActiveState=inactive
+$systemctl enable NetworkManager; systemctl start NetworkManager
+```
+
+Once the installation is done, you can see the following output.
+
+```
+  PLAY RECAP *********************************************************************
+  20.10.45.111               : ok=383  changed=142  unreachable=0    failed=0
+  20.10.45.112               : ok=61   changed=13   unreachable=0    failed=0
+  localhost                  : ok=14   changed=0    unreachable=0    failed=0
+  INSTALLER STATUS *****************************************************************************************************************************************************
+  Initialization             : Complete (0:00:58)
+  Health Check               : Complete (0:05:35)
+  etcd Install               : Complete (-1 day, 22:57:25)
+  NFS Install                : Complete (0:00:48)
+  Master Install             : Complete (0:06:25)
+ Master Additional Install  : Complete (0:01:01)
+ Node Install               : complete (0:03:20)
+```
+
+-The install process takes approximately 10-15 minutes.
+
+### Start OpenShift Container Platform
+After a successful install, use the following command to start OpenShift Container Platform.
+
+```
+systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
+```
+
+-Before you do anything else, log in at least one time with the default system:admin user, on the master run the following command.
```

Comment:
anything more, log in at least once with the default...user, and on the master...